src/directives/class-map.ts

```diff
@@ -48,7 +48,7 @@ export const classMap = directive((classInfo: ClassInfo) => (part: Part) => {
 
   // handle static classes
   if (!classMapCache.has(part)) {
-    element.className = committer.strings.join(' ');
+    element.setAttribute('class', committer.strings.join(' '));
```

Comment:
Oh, we could actually save some file size here. `classList` has a getter/setter `value` property which acts the just like `element.className` but will work for both SVG and HTML Elements.

Reply:
Good catch! Changed.

Reply:
> classList has a getter/setter value
AFAIK IE 11 does not supports it

I'm all for dropping IE11 support but...

Reply:
Ugh, you're right.